actor/src/main/scala/org/apache/pekko/pattern/BackoffOptions.scala

```diff
@@ -255,6 +258,22 @@ object BackoffOpts {
       maxBackoff: java.time.Duration,
       randomFactor: Double): BackoffOnStopOptions =
     onStop(childProps, childName, minBackoff.asScala, maxBackoff.asScala, randomFactor)
+
+  /**
+   * Calculates an exponential back off delay.
+   */
```

Comment:
could you add the `@since 1.2.0` here?

Reply:
I see, totally missed that.

Reply:
Do you think we should put it in RetrySupport or here? @pjfanning